samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
```

Comment:
my intention is to only exercise a test config when user finishes the editing - thus when textFieldEditDidEnd, not UIControlEventEditingChanged which TestEnabledUITextField supports. So i don't want to execute unnecessary tests in manual mode (imaging user typing in font size 7 and 8 and tests executed for 7, and 78, which is not what i want. I only want test execute for 78 when user exit editing.  In auto execution mode (functional test), the test itself enumerate all config and after finishing last config (e.g., fontSize), and tests knows it can start to execute the API call or start verification since all configs have been set and it does not have to reply on any events to trigger verification.

Does this make sense? 

---
In reply to: [104793428](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r104793428) [](ancestors = 104793428)